android/src/main/java/com/bitmovin/player/reactnative/RNSubtitleViewManager.kt

```diff
@@ -0,0 +1,136 @@
+package com.bitmovin.player.reactnative
+
+import android.util.TypedValue
+import com.bitmovin.player.SubtitleView
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.uimanager.SimpleViewManager
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.annotations.ReactProp
+
+private const val MODULE_NAME = "BitmovinSubtitleView"
+
+@ReactModule(name = MODULE_NAME)
+class RNSubtitleViewManager(private val context: ReactApplicationContext) : SimpleViewManager<SubtitleView>() {
+
+    /**
+     * Exported module name to JS.
+     */
+    override fun getName() = MODULE_NAME
+
+    /**
+     * The component's native view factory. RN calls this method multiple times
+     * for each component instance.
+     */
+    override fun createViewInstance(reactContext: ThemedReactContext) = SubtitleView(reactContext)
+
+    @ReactProp(name = "playerId" )
+    fun setPlayerId(view: SubtitleView, playerId: String?) {
+        val player = context.getNativeModule(PlayerModule::class.java)?.getPlayer(playerId)
+        if (player != null) {
+            view.setPlayer(player)
+        }
+    }
+
+    /**
+     * Sets whether font sizes embedded within the cues should be applied.
+     * Enabled by default.
+     * Only takes effect if setApplyEmbeddedStyles is set to true.
+     */
+    @ReactProp(name = "applyEmbeddedFontSizes", defaultBoolean = true )
+    fun setApplyEmbeddedFontSizes(view: SubtitleView, applyEmbeddedFontSizes: Boolean) {
+        view.setApplyEmbeddedFontSizes(applyEmbeddedFontSizes)
+    }
+
+    /**
+     * Sets whether styling embedded within the cues should be applied.
+     * Enabled by default.
+     * Overrides any setting made with setApplyEmbeddedFontSizes.
+     */
+    @ReactProp(name = "applyEmbeddedStyles", defaultBoolean = true )
```

Comment:
I'm not familiar enough with the Bitmovin Android SDK, but it seems that the `applyEmbeddedStyles` is mutually exclusive with `userDefaultStyle`.  I think it may make more sense to provide an enum of a single property that sets the style, and font size appropriately.  What do y'all think?